src/main/java/org/sonarsource/scanner/maven/SonarQubeMojo.java

```diff
@@ -120,6 +122,12 @@ public void execute() throws MojoExecutionException, MojoFailureException {
     }
   }
 
+  private void verifyMavenVersion() throws MojoExecutionException {
+    if (!runtimeInformation.isMavenVersion("3.0")) {
+      throw new MojoExecutionException("Maven 2 is not supported anymore by sonar-scanner-maven. Please use Maven 3");
```

Comment:
I would replace "sonar-scanner-maven" either by "sonar-maven-plugin" (artifact ID) or by "the SonarQube Scanner for Maven"